Tasks/Common/nuget-task-common/LocationHelpers.ts

```diff
@@ -103,9 +114,9 @@ export function getConnectionDataForArea(
     serviceUri: string,
     areaName: string,
     areaId: string,
+    username: string,
     accessToken: string): Q.Promise<locationApi.ConnectionData> {
```

Comment:
consider rename: really 'password', which happens to be an access token

Reply:
cool, will do.